src/spark-connect-script/build.rs

```diff
@@ -0,0 +1,28 @@
+fn main() -> std::io::Result<()> {
```

Comment:
let place this crate under filepaths:

`src/scripts/spark-connect-generation`

Reply:
ab6c56ba978e84a37b65077b368bd408446e25c8